LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2014-2015 Fabien Potencier
```

Comment:
@javiereguiluz I'm not completely sure here since afaik you initially wrote the first version of the installer.